src/unstract/adapters/x2text/llm_whisperer/src/static/json_schema.json

```diff
@@ -12,13 +12,26 @@
       "default": "",
       "description": "Provide a unique name for this adapter instance. Example: LLM Whisperer 1"
     },
-
     "url": {
       "type": "string",
       "title": "URL",
       "format": "uri",
       "default": "http://unstract-llm-whisperer:3005",
       "description": "Provide the URL of the LLM Whisperer server."
+    },
+    "extraction_mode": {
+      "type": "string",
+      "title": "Extraction Mode",
+      "enum": ["text", "ocr"],
+      "default": "text",
+      "description": "Text mode tries to extract text from PDF and falls to OCR if the PDF is a scanned image PDF. This should be your default selection. Use OCR mode if you want to force OCR to extract text. This could be useful if you are dealing with malformed PDFs."
```

Comment:
Did you check, How this diplayes in UI. If this descption larger maay some brakes in UI .

Reply:
It does render fine
![image](https://github.com/Zipstack/unstract-adapters/assets/117059509/9acdd6b7-a341-41bc-b8c6-542c6034b62a)